### PR TITLE
[1] Clean up the codebase from the uniswap overrides

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Clean up the codebase from the uniswap overrides.

Removed:
 - `custom/assets`
 - `custom/components`
 - `custom/constants`
 - `custom/lib`
 - `custom/serviceWorker`
 - `custom/tokens`
 - `custom/types`
 
## What changed
1. Unused files are removed
2. Files from `src/custom/*` are moved to `src/*`

## How `custom` files were moved to `src`
There are two cases:
1. When `moduleMod.tsx` contains `export * from '@src/module.tsx'`
2. When it doesn't

The second case is the simplest. In this case, I just moved files from `src/custom/module` to `src/module`.
In the first case, I manually merged the original and `custom` files.

### CODE REVIEW

**Review by alfetopito — APPROVED** [review]
🤞

**Review by anxolin — APPROVED** [review]
Great! I couldn't review all the details, since I don't have 720h free today. But high level seems good. Would be nice to test with Elena. If you have high confidence and we fear the conflicts, we can merge earlier and address any issues in follow ups. 

But at least, lets make some good tests on the main flows are no broken

File: src/components/Header/Polling.tsx [hunk 1]
```diff
@@ -137,7 +195,7 @@ export default function Polling() {
   //TODO - chainlink gas oracle is really slow. Can we get a better data source?
 
   return (
-    <>
+    <Wrapper>
```
[the remark this answers is not in the document]
Comment (shoom3301) [reply src/components/Header/Polling.tsx:198]: Other exports from `export * from '@src/components/Header/Polling'` are not used

[the remark this answers is not in the document]
Comment (anxolin) [reply src/components/Header/Polling.tsx:198]: 👍

**Pull request merged**